src/server/game/Entities/Player/Player.h

```diff
@@ -967,6 +967,8 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         void SetPvPDeath(bool on) { if (on) m_ExtraFlags |= PLAYER_EXTRA_PVP_DEATH; else m_ExtraFlags &= ~PLAYER_EXTRA_PVP_DEATH; }
 
         void GiveXP(uint32 xp, Unit* victim, float group_rate=1.0f);
+        uint32 GetPlayerXP() const { return GetUInt32Value(PLAYER_XP); }
+        void SetPlayerXP(uint32 newXP) { SetUInt32Value(PLAYER_XP, newXP); }
```

Comment:
This already exists in master branch and is named `SetXP`, some consistency would be great (even if it did not exist, the field is named `XP`, not `PLAYER_XP`, `PLAYER_` is just a prefix for enum value)